mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/bridge/builtin/impl/DefaultJavaUtilDateValueBridge.java

```diff
@@ -27,9 +27,9 @@ public Instant toIndexedValue(Date value, ValueBridgeToIndexedValueContext conte
 			return null;
 		}
 
-		// java.sql.Date and java.sql.Time do not support toInstant(). See HSEARCH-3670.
+		// java.sql.* types do not support toInstant(). See HSEARCH-3670.
 		// We're using here the same strategy applied on DefaultJavaSql*ValueBridge to handle them.
-		return ( value instanceof java.sql.Date ) | ( value instanceof java.sql.Time ) ?
+		return ( value instanceof java.sql.Date ) || ( value instanceof java.sql.Time ) || ( value instanceof java.sql.Timestamp ) ?
```

Comment:
Same comment as before about always using `Instant.ofEpochMilli( value.getTime() )`.

Also, you didn't test `Timestamp`?

Reply:
It seems that it is not possible to add a test for a timestamp without a significant change. 

The `FieldDefaultBridgeIT#projection` method test would fail, since `DefaultJavaUtilDateValueBridge#fromIndexedValue` method cannot distinguish the cases, it converts the indexed `Instant` value to a `util.Date` instance and, unlike `java.sql.Date` and  `java.sql.Time`, the equals  of `Timestamp` check the instance type.

Does such a test worth the effort to handle it?